build.sh

```diff
@@ -23,7 +23,7 @@ sudo apt-get install -y \
 
 sudo mkdir /usr/local/php7
 
-git clone https://github.com/php/php-src.git
+git clone -b PHP-7.0.0 --depth=1 https://github.com/php/php-src.git
 cd php-src
 git checkout PHP-7.0.0
```

Comment:
PHP-7.0.1 is the latest and with `-b` within the `clone` the checkout isn't  important anymore